packages/relay/src/lib/eth.ts

```diff
@@ -1161,12 +1161,19 @@ export class EthImpl implements Eth {
       txSubmitted = true;
       // Wait for the record from the execution.
       let txId = contractExecuteResponse.transactionId.toString();
-      const formattedId = formatTransactionId(txId);
+      const formattedId = formatTransactionIdWithoutQueryParams(txId);
+      
+      // handle formattedId beng null
+      if (!formattedId) {
+        throw predefined.INTERNAL_ERROR(`Invalid transactionID.`);
```

Comment:
Let's add a log here and generalize the error so as to
```suggestion        
        throw predefined.INTERNAL_ERROR(`Invalid transactionID: ${txId}`);
```

Reply:
Heard!